audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/model/AuditEventName.java

```diff
@@ -1,5 +1,25 @@
 package com.sequenceiq.cloudbreak.audit.model;
 
 public enum AuditEventName {
-    DATAHUB_CLUSTER_CREATION
+    CREATION_DATAHUB_CLUSTER,
+    DELETION_DATAHUB_CLUSTER,
+    STOPPAGE_DATAHUB_CLUSTER,
+    RESIZING_DATAHUB_CLUSTER,
+    MANUAL_REPAIR_DATAHUB_CLUSTER,
+    SYNC_DATAHUB_CLUSTER,
+    RETRY_DATAHUB_CLUSTER,
+    INSTANCE_DELETION_DATAHUB_CLUSTER,
+    MAINTENANCE_DATAHUB_CLUSTER,
+    STARTING_DATAHUB_CLUSTER,
```

Comment:
Please rename the corresponding constants to the following, as it was recommended in the event names sheet:

CREATE_DATAHUB_CLUSTER
DELETE_DATAHUB_CLUSTER
STOP_DATAHUB_CLUSTER
RESIZE_DATAHUB_CLUSTER
DELETE_DATAHUB_CLUSTER_INSTANCE
MAINTAIN_DATAHUB_CLUSTER
START_DATAHUB_CLUSTER

Reply:
done